frontends/pytorch/test/ivalue_import/annotations/export-recursive.py

```diff
@@ -41,8 +41,8 @@ def forward(self):
 # CHECK:           torch.method private "not_exported_method", @{{.*}}
 # CHECK:         }              
 annotator.exportNone(class_type)
-annotator.exportPath(['s', 'exported'], class_type)
-annotator.exportPath(['s', 'forward'], class_type)
+annotator.exportPath(class_type, ['s', 'exported'])
```

Comment:
This is making my eyes bleed. I know you want to sugar it later, but wdyt about adding an overload to `exportPath` that takes a string arg and splits on dots?

(also nit: these should probably be snake-case)

Reply:
Is there an easy way to add that in Python? I don't want to pollute the C++/Python interface with sugar-only methods.

I could also switch the methods to always take a dotted string. I guess that would be robust enough too. WDTY?

Reply:
Will fix in a future patch.